### PR TITLE
Run nbqa-black formatter on BSM

### PR DESCRIPTION
**Pull request by MarcoGorelli**
# Thank you for opening a pull request!

Please check our [style guide](https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style), and also make sure that the notebooks you've modified pass the `pre-commit` checks. If, for example, you modified `notebook1.ipynb` and `notebook2.ipynb`, you could do this by running:

```bash
pre-commit run --files notebook1.ipynb notebook2.ipynb
```

You may need to run it a couple of times before all the checks pass.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     -   id: check-toml
     -   id: check-yaml
 - repo: https://github.com/nbQA-dev/nbQA
-  rev: 0.5.4
+  rev: 0.5.6
   hooks:
     - id: nbqa-black
       additional_dependencies: [black==20.8b1]
-      files: ^Rethinking_2/
+      files: ^(Rethinking_2|BSM)/
```
Comment (AlexAndorra) [inline .pre-commit-config.yaml:12]: Out of curiosity, why do you need to change this? Wasn't the previous pattern more permissive?

**Review by AlexAndorra — APPROVED** [review]
Thanks @MarcoGorelli ! I just had a small question -- will merge afterwards 😉

File: .pre-commit-config.yaml (hunk 1 above)
Comment (MarcoGorelli) [reply .pre-commit-config.yaml:12]: `files` specifies which files to run a formatter. It takes a regular expression which is matched with `re.search`. So before, `^Rethinking_2/` would only have matched the `Rethinking_2` folder. The new one matches the `BSM` one as well, and so is less restrictive. The idea is to make the pattern less and less restrictive until it covers all files

Comment (AlexAndorra) [reply .pre-commit-config.yaml:12]: Ah ok, I had it backwards -- I though the previous one was recursive. All good then, thanks!

**Pull request merged**